blog: v6 announcement (#4598)
simplex-chat/simplex-chat: blog/20240814-simplex-chat-vision-funding-v6-private-routing-new-user-experience.md

---
layout: layouts/article.html
title: "SimpleX Chat: vision and funding, v6.0 released with the new user experience and private message routing."
date: 2024-08-14
# image: images/20230422-video.png
# imageBottom: true
# previewBody: blog_previews/20230422.html
draft: true
permalink: "/blog/20240814-simplex-chat-vision-funding-v6-private-routing-new-user-experience.html"
---

# SimpleX network: vision and funding, v6.0 released with the new user experience and private message routing.

**Published:** Aug 14, 2024

[SimpleX Chat: vision and funding 2.0](#simplex-chat-vision-and-funding-20):
- [The past](#the-past).
- [The present](#the-present).
- [The future](#the-future).

[What's new in v6.0](#whats-new-in-v60):
- [Private message routing](#private-message-routing) - now enabled by default
- [New chat experience](#new-chat-experience)
  - [Connect to your friends faster](#connect-to-your-friends-faster).
  - [Archive contacts to chat later](#archive-contacts-to-chat-later).
  - [New way to start chat](#new-way-to-start-chat).
  - [Reachable chat toolbar](#reachable-chat-toolbar).
  - [Moderate like a pro](#moderate-like-a-pro).

>   - [Increase font size](#increase-font-size)<sup>*</sup>.

Reviewer: + themes on ios

- [New media options](#new-media-options)
  - [Play from the chat list](#play-from-the-chat-list).
  - [Blur for better privacy](#blur-for-better-privacy).
  - [Share from other apps](#share-from-other-apps)<sup>**</sup>.
- [Improved networking and reduced battery usage](#improved-networking-and-reduced-battery-usage)

\* Android and desktop apps.

> \*\* iOS app.

Reviewer: ```suggestion
\*\* New for iOS app.
```

## SimpleX Chat: vision and funding 2.0

### The past

...

### The present

...

### The future

...

## What's new in v6.0

v6.0 is one of our biggest releases ever, with a lot of focus on UX and stability improvements, and the new features the users asked for. 

### Private message routing

The private message routing [we announced before](./20240604-simplex-chat-v5.8-private-message-routing-chat-themes.md) is now enabled for all users by default – it protects users IP addresses and sessions from the destination servers.

### New chat experience

#### Connect to your friends faster 

> This version includes messaging protocol improvements that cut down the number of messages requried for two users to connect twice. Not only it means connecting faster and using less traffic, this change allows to start sending messages sooner than before - so you would see "connecting" in the list of the chats for a much shorter time than before.

Reviewer: First time I read it "connect twice" and it took me a second to figure out "twice" refers to "cut down". I'd even sacrifice grammar a little and say "that reduce by half the number of messages" (is it even incorrect?) to make it more readable

```suggestion
This version includes messaging protocol improvements that reduce the number of messages required for two users to connect by half. Not only it means connecting faster and using less traffic, this change allows to start sending messages sooner than before - so you would see "connecting" in the list of the chats for a much shorter time than before.
```

Reviewer: also, 2x "than before"

> It will be improved further in the next version - you will be able to send messages straight after using the link, without waiting for your contact to be online.

Reviewer: sounds to good to be true without explanation - one may question what's happening behind the scenes. maybe give a link to some rfc, explanation that in some cases this is achievable because connection is "ready in one direction"?

#### Archive contacts to chat later

Now you have two new options when deleting a conversation:

> - only delete conversation, and archive contact - we will add archiving conversation without clearing it in the next version.

Reviewer: unclear why we didn't do it initially - "was it that hard?"
```suggestion
- only delete conversation, and archive contact - we will add archiving conversation without clearing it in the next version, as a popular request from beta testers.
```

- delete contact but keep the conversation.

> Also, deleting a contact now requires double confirmation, so you are less likely to delete the contact accidentally.

Reviewer: I don't think it's a good thing - rather a necessity because of abundance of options. if we decide to separate archive into two variants, and delete into two variants, we can go back to a single less annoying confirmation (it would still require user 3 interactions - choose delete in menu/swipe, process "keep conversation" toggle, tap notify or not, yet another confirmation when it's unnecessary is next level evil).

Reviewer: at least I wouldn't advertise it as feature now so that we can discuss/think a bit

Author: lots of places ask for double confirmation of irreversible deletions, don't think it should be removed. In any case, it's not a problem to write about it.

#### New way to start chat

Now when you tap pencil button, you will see a large New message sheet, that adds to the options you had before some new functions.

Old options:
- *Add contact* to create a new 1-time invitation link,

> - *Scan/paste link*: to use the link you received, it can be 1-time invitation, a public SimpleX address, or a link to join the group.

Reviewer: ```suggestion
- *Scan/Paste link*: to use the link you received, it can be 1-time invitation, a public SimpleX address, or a link to join the group.
```

This layout is enabled by default, and you can disable it right from the list of chats when you install the new version if you prefer to use conventional UI.

But give it a try – our experience is that that after a day of using it, it seems to be the only right way. You can always toggle it in the Appearance settings.

#### Moderate like a pro

As much as we disagree with attacks on the freedom of speech on the society level – various groups must be able to express their opinions, be they far left, far right, or anybody in between – we also believe that in the small communities, community owners have full discretion to decide which content is allowed and which is not. But as communities grow, bad actors begin to join in order to disrupt, subvert and troll the conversations. So, the moderation tools are critical for small public communities to thrive. 

SimpleX Chat already has quite a few moderation tools available for community owners:
- Moderate individual messages.

> - Set default role for the new members to join to "observer" - they won't be able to send messages until you allow them. In addition to that, by enabling default messages for admins and owners only you can reach out to the new members and ask them some questions before allowing them to send messages.

Reviewer: ```suggestion
- Set default role for the new members to "observer" - they won't be able to send messages until you allow them. In addition to that, by enabling default messages for admins and owners only you can reach out to the new members and ask them some questions before allowing them to send messages.
```

Also, this version makes profile images of the blocked members blurred, to prevent the abuse via inappropriate profile images.

#### Increase font size

Android and desktop apps now allow to increast font size inside the app, without changing the system settings. Desktop in addition to that allows to zoom the whole screen - it can be helpful on some systems with a limited support of high density displays.

These settings can be changed via appearance settings.

### New media options

#### Play from the chat list

Now you can interact with the media in the last message directly from the list of the chats.

This is very convenient – when somebody sends you a voice message or a video, they can be played directly from the list of chats, without opening a conversation. Similarly, an image can be opened, a file can be saved, and the link with preview can be opened in the browser.

And, in some circumstances, this is also more private, as you can interact with the media, without opening the whole conversation.

We will add the option to return missed calls from the chat list in the next version.

#### Blur for better privacy

> You can set all images and videos to blur in your app, and unblur them on tap or on hover in desktop app. The blur level can be set in Privacy and security settings.

Reviewer: second "app" makes it harder to process
```suggestion
You can set all images and videos to blur in your app, and unblur them on tap (or on hover in desktop app). The blur level can be set in Privacy and security settings.
```

#### Share from other apps

Not much to brag about, as most iOS apps have it, and users expect it to be possible.

But iOS makes it much harder to develop the capability to share into the app than Android, so it's only in this version you can share images, videos, files and links into SimpleX Chat from other apps.

### Improved networking and reduced battery usage

> This version includes the statistics of how your app communicates with all servers when sending and receipving messages and files. This information also includes the status of connection to all servers from which you receive messages - whether the connection is authorized to push messages from server to your device, and the share of these active connections.

Reviewer: ```suggestion
This version includes the statistics of how your app communicates with all servers when sending and receiving messages and files. This information also includes the status of connection to all servers from which you receive messages - whether the connection is authorized to push messages from server to your device, and the share of these active connections.
```

> Please note, that when you send a message to a group, your app has to send it to each member separately, so sent message statistics accounts for that - it may seem to be quite a large number if you actively participate in some large groups. Also, message counts not only include visible messages you receive and send, but also any service messages, reactions, message updates, message deletions, etc. - this is the correct reflection of how much traffic your app uses.

Reviewer: ```suggestion
Please note, that when you send a message to a group, your app has to send it to each member separately, so sent message statistics account for that - it may seem to be quite a large number if you actively participate in some large groups. Also, message counts not only include visible messages you receive and send, but also any service messages, reactions, message updates, message deletions, etc. - this is the correct reflection of how much traffic your app uses.
```
